app/views/api/v2/settings/main.json.rabl

```diff
@@ -3,3 +3,7 @@ object @setting
 extends "api/v2/settings/base"
 
 attributes :value, :description, :category, :settings_type, :default, :created_at, :updated_at
+
+node do |s|
```

Comment:
Nitpick, prefer `node :category_name` syntax instead of a hash if adding just one node.